experimental/origin-admin/src/queries/UserOffers.js

```diff
@@ -1,5 +1,4 @@
 import gql from 'graphql-tag'
-import fragments from './Fragments'
```

Comment:
Did we need to get rid of this?

Reply:
No fragments are used in this file. Should we still keep it in?

Reply:
There used to be a fragment on line 29... any reason it was removed?

Reply:
I might be missing something here. Does a fragment definition serve any purpose if it is not referenced inside the query? 
Example: 
```
nodes {
  id
  listingId
  ... basicListingFields
}
```

Since there was no `...basicListingFields` I though I should just remove it?

Reply:
Ahhh yea you're right, my bad :)